frontend/src/routes/SearchPage/searchDefinitions.tsx

```diff
@@ -1360,10 +1360,13 @@ export function CreateDetailsLink(item: any) {
     switch (item.kind) {
         case 'cluster':
             return <a href={`/multicloud/clusters/${item.name}/overview`}>{item.name}</a>
+
         case 'application':
-            if (item.apigroup === 'app.k8s.io') {
+            if (item.apigroup === 'app.k8s.io' || item.apigroup === 'argoproj.io') {
                 // only redirect to apps page if it is an ACM application
-                return <a href={`/multicloud/applications/${item.namespace}/${item.name}`}>{item.name}</a>
+
+                return <a href={`/multicloud/applications/${item.namespace}/${item.name}?apiVersion=${item.apigroup}%2F${item.apiVersion}$cluster=${item.cluster}`}>{item.name}</a>
```

Comment:
`item.apiversion` (all lowercase)

**Tip:** You can use the browser's developer tools to inspect the item's properties.
```suggestion
                return <a href={`/multicloud/applications/${item.namespace}/${item.name}?apiVersion=${item.apigroup}%2F${item.apiversion}$cluster=${item.cluster}`}>{item.name}</a>
```
![image](https://user-images.githubusercontent.com/4671325/118580038-26755f80-b75d-11eb-8eff-5c203bb9146d.png)